bigchaindb/commands/utils.py

```diff
@@ -14,41 +15,63 @@
 from bigchaindb.version import __version__
 
 
-def start_rethinkdb():
+def start_temp_rethinkdb(port=0, directory=None):
+    directory = directory or tempfile.mkdtemp()
+
+    extra_opts = ['--cluster-port', '0',
```

Comment:
This function is to run a temporary RethinkDB to test things, and the default port can be in use. With port `0` you leave the choice of the port to the OS.